src/Chapter2.hs

```diff
@@ -848,7 +886,10 @@ and reverses it.
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
+rewind :: [a] -> [a]
+rewind xs = go xs []
+  where go [] accum = accum
+        go (y:ys) accum = go ys (y : accum)
```

Comment:
Nice solution! Now you're ready for interview in Haskell, if they ask you to implement a list reverse function :smirk: